File: 2021/template.py
test = list([n for n in open('dayXX.test').read().splitlines()])
data = list([n for n in open('dayXX.data').read().splitlines()])


def part1(lines):
    return 0


p1 = part1(test)
print(f'test1: {p1} - {p1 == 1}')
p1 = part1(data)
print(f'real1: {p1} - {p1 == 1}')


def part2(lines):
    return 0


p2 = part2(test)
print(f'test1: {p2} - {p2 == 1}')
p2 = part2(data)
print(f'part2: {p2} - {p2 == 1}')

File: template.go
package main

import (
	"fmt"
	"github.com/jjbeto/adventofcode/aoc"
)

func part1(lines []string) int {
	return len(lines)
}

func part2(lines []string) int {
	return len(lines)
}

func main() {
	numbers := aoc.ReadLines("202X/dayXX", "test.txt", aoc.ToString)
	fmt.Println("part1", part1(numbers))
	fmt.Println("part2", part2(numbers))
}
